resources/sdk/pureclouddotnet/scripts/publishNuget.sh

```diff
@@ -44,15 +44,15 @@ echo '<?xml version="1.0" encoding="utf-8"?>'\
 '			<dependency id="Newtonsoft.Json" version="11.0.2" />'\
 '			<dependency id="RestSharp" version="110.2.0" />'\
 '			<dependency id="ini-parser" version="2.5.2" />'\
-'			<dependency id="System.Text.Json" version="6.0.0" />'\
+'			<dependency id="System.Text.Json" version="7.0.2" />'\
```

Comment:
I will call this change out specifically, updating a dependency by a major version during a refactor is probably a bad idea.

Reply:
Well, I wonder some of the changes popped up. These are all merged previosuly. I resolved the merge conflicts. Lot of the functional changes that you commented were there in seperate PRs. Thanks Tim for pointing this out.